lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -718,5 +718,16 @@ fb_actions.sort_by_date = function(prompt_bufnr)
   end)
 end
 
+--- If the prompt is empty, goes up to parent dir. Otherwise, acts as normal.
+fb_actions.backspace = function(prompt_bufnr)
```

Comment:
Can we make this function also take a `bypass` arg so we're not defaulting to `true` for `goto_parent_dir(prompt_bufnr, true` below?

Reply:
You got it :)